src/components/GameStatusLabel.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const GameStatusLabel = props => {
+    if (props.gameStatus === 'start') {
```

Comment:
Consider replacing these with `switch` and creating an enum for GameStatus as well